docs/manual/install/index.rst

```diff
@@ -111,3 +111,32 @@ Stable versions of Qtile can be installed from PyPI:
 As long as the necessary libraries are in place, this can be done at any point,
 however, it is recommended that you first install xcffib to ensure the
 cairo-xcb bindings are built (see above).
+
+Wayland
```

Comment:
It may be good to note the current key limitations if there are things we know will be very broken at the moment.

Reply:
Yep sounds good. The main ones are Z positoning and the status bar, so I've added those.